### ISSUE TITLE
Parse parameters from Intent

### ISSUE THREAD

**Issue by brucezz** [nekocode/Meepo#4]
Can this library parse parameters from `Intent` automatically ?

Right now you need to parse parameters manually, like `getIntent().getStringExtra("title")`  in sample code.

😄

**Comment by nekocode** [comment]
No, it can't now. You need to get the extra data from Intent manually.

**Comment by brucezz** [comment]
Do you have any plan to support this feature ?

Similar to [ARouter](https://github.com/alibaba/ARouter), call `ARouter.getInstance().inject(this)`, then parameter injection will apply automatically.

Anyway, Meepo is a interesting and useful library. Thanks.

**Comment by nekocode** [comment]
I don't think it's a necessary feature. Because Meepo is not only a Activity router, it supports the activity routing by the provided GotoActivityAdapter. So I don't think parsing the Intent parameters is the job of it.

But i will consider it. Maybe I will let it be supported by providing an extra library in this repository.
